seed/templates/seed/pm_integration.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>SEED's Portfolio Manager Integration</title>
+</head>
+<body>
+<h1>Enter Portfolio Manager Credentials</h1>
+<form action="/pm_integration_worker/" method="post">
+  {% csrf_token %}
+  Email:<br>
+  <input type="text" name="email" value="a@a.a"><br><br>
+  Username:<br>
+  <input type="text" name="username" value="SEED_City_Test"><br><br>
+  Password:<br>
+  <input type="password" name="password" value=""><br><br>
+  Report Template Name:<br>
+  <input type="text" name="template_name" value="SEED City Test Report"><br><br>
+  <input type="submit" value="Submit">
+</form>
+</body>
+</html>
```

Comment:
Right now this is just a standalone file, but I would guess ultimately it would be a dialog...

Reply:
Almost ready to get rid of this page once the modal becomes slightly more functional.